scripts/resource-autogen/generated/servicemappings/tags.yaml

```diff
@@ -91,3 +91,30 @@ spec:
           targetField: name
           valueTemplate: "tagKeys/{{value}}"
           parent: true
+    - name: google_tags_location_tag_binding
+      kind: TagsLocationTagBinding
+      autoGenerated: true
+      idTemplate: "{{location}}/{{name}}"
+      idTemplateCanBeUsedToMatchResourceName: false
+      resourceAvailableInAssetInventory: false
+      serverGeneratedIDField: name
+      resourceID:
+        targetField: name
+      resourceReferences:
+        - key: parentRef
+          tfField: parent
+          gvk:
+            kind: Project
```

Comment:
Per the [list](https://cloud.google.com/resource-manager/docs/tags/tags-supported-services), Project is one of the supported resource, so it's fine to have it here. Just a side note that you are able to add multiple resource kinds like [example code](https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/master/config/servicemappings/compute.yaml#L1402-L1432). Considering there're numerous supported resources, I won't suggest add all of them. 

I would recommend always using "external" instead of "name" for parentRef field, in case you want to reference to other kinds of resources that not listed in the servicemapping. i.e. in your example, 
```
parentRef:
    **external**: //storage.googleapis.com/projects/_/buckets/<bucket name>
```

Another alternative would be simply remove the reference of parent field, and use plain text directly. i.e.
```
parent: "your_value"
```
You'll need to remove parentRef from servicemapping file of course. Since this is Alpha, I think we can add the reference back when needed. However, I haven't tested this and I'm unsure if this is going to work as expected, if you feel this is clearer and easier, feel free to try this way.

Reply:
I'm fine to approve this PR at this point. Make sure that always use "external" for parentRef:https://cloud.google.com/config-connector/docs/how-to/creating-resource-references#external_references